poetry.lock

```diff
@@ -1619,21 +1619,23 @@ files = [
 widechars = ["wcwidth"]
 
 [[package]]
-name = "terra-scientific-pipelines-service-api-client"
-version = "0.1.36"
+name = "teaspoons-client"
```

Comment:
i assume this'll get updated in pyproject.toml - basically we can ignore these poetry.lock changes for now till the service PR is merged and you update this

Reply:
yup yup thats what i expect as well